ngx-fudis/projects/ngx-fudis/src/lib/components/form/form/form-dynamic-validators.stories.ts

```diff
@@ -193,7 +193,7 @@ import { FudisGroupValidators } from '../../../utilities/form/groupValidators';
                 </fudis-checkbox-group>
               </fudis-grid>
               <fudis-grid [columns]="{ md: 'inputLg auto' }">
-                <hr fudisGridItem [columns]="'stretch'" class="fudis-hr" aria-hidden="true" />
+                <fudis-divider fudisGridItem [columns]="'stretch'" />
```

Comment:
This divider could be moved in between this and the previous grid (like all the other dividers in this view), therefore it doesn't need any additional attributes.